src/main/java/dev/hephaestus/glowcase/Glowcase.java

```diff
@@ -2,16 +2,8 @@
 
 import com.google.common.base.Supplier;
 import com.google.common.base.Suppliers;
-import dev.hephaestus.glowcase.block.HyperlinkBlock;
-import dev.hephaestus.glowcase.block.ItemDisplayBlock;
-import dev.hephaestus.glowcase.block.PopupBlock;
-import dev.hephaestus.glowcase.block.SpriteBlock;
-import dev.hephaestus.glowcase.block.TextBlock;
-import dev.hephaestus.glowcase.block.entity.HyperlinkBlockEntity;
-import dev.hephaestus.glowcase.block.entity.ItemDisplayBlockEntity;
-import dev.hephaestus.glowcase.block.entity.PopupBlockEntity;
-import dev.hephaestus.glowcase.block.entity.SpriteBlockEntity;
-import dev.hephaestus.glowcase.block.entity.TextBlockEntity;
+import dev.hephaestus.glowcase.block.*;
+import dev.hephaestus.glowcase.block.entity.*;
```

Comment:
Your IDE doesn't appear to be respecting [this line](https://github.com/ModFest/glowcase/blob/1.21/.editorconfig#L15) in the editor config intended to prevent automatic star imports. Could you remedy these?

Reply:
I don't know why Intellij is not respecting that editor config line but I've fixed it